physical/requalifier.go

```diff
@@ -44,3 +45,10 @@ func (node *Requalifier) Metadata() *metadata.NodeMetadata {
 
 	return metadata.NewNodeMetadata(node.Source.Metadata().Cardinality(), eventTimeField)
 }
+
+func (node *Requalifier) Visualize() *graph.Node {
+	n := graph.NewNode("Requalifier")
+	n.AddField("qualifier", node.Qualifier)
+	n.AddChild("source", node.Source.Visualize())
```

Comment:
Same

Reply:
Here you have a field and child they'll be in different places in the node.